### DISCUSSION TITLE
Extended search syntax only on staging

### DISCUSSION THREAD

**Discussion by finestructure** [SwiftPackageIndex/SwiftPackageIndex-Server#1396] [Q&A]
I've just noticed that we've got Jame's initial search extension only enabled on staging:

<img width="755" alt="CleanShot 2021-11-25 at 09 39 05@2x" src="https://user-images.githubusercontent.com/65520/143407607-b830d745-ac74-4fc9-a68d-a8291c80ef11.png">

Prod:

<img width="399" alt="CleanShot 2021-11-25 at 09 38 46@2x" src="https://user-images.githubusercontent.com/65520/143407627-d5dd283e-2b40-4ad6-a877-d29244f5a9c8.png">

Was there a reason we didn't ship it? I don't remember but I can't really see a reason not to?

**Comment by finestructure** [comment]
For reference, the PR was #1226 and the feature flag is set in `SearchFilter.swift:37`.

**Comment by Sherlouk** [comment]
Intentional, it's pending the UI work so that the search results page is a bit nicer and shows the filters which have been applied.

It does technically work already but want to tidy up the UI. Still on my TODO

**Reply by Sherlouk** [reply]
#1228

**Reply by finestructure** [reply]
I mean I guess we could style the special search term a bit but I don't think we'd need to feature flag that. If the `stars:>1000` were somehow dropped from the search term and you had results filtered out unknowingly that'd be a different story of course. But as such it's all there, isn't it?

**Reply by daveverwer** [reply]
I'm still happy to split the work on this and take care of the front end and styling work to get it implemented. I'd love to have the whole feature before we launch it though, it'll make for a much better experience and enable a blog post etc...
